petrel/auth.go

```diff
@@ -46,7 +46,7 @@ func newAuthServer(serverIP string, port int, vpnnet string) (*AuthServer, error
 
 	internalIP, ipNet, err := net.ParseCIDR(vpnnet)
 	if err != nil {
-		fmt.Println("Error in vpnnet format: %V", vpnnet)
+		log.Errorf("Invalid vpnnet format: %v", vpnnet, err)
```

Comment:
The number of format arguments and the placeholders don't match. Does it matter? `fmt.Printf ` won't work in this case. I don't know what will happen for `go-logging`.